service.py

```diff
@@ -37,9 +38,69 @@ def autostart():
         else:
             cleaner.monitor.waitForAbort(service_sleep)
 
-    log("Abort requested. Terminating.")
+    debug("Abort requested. Terminating.")
     return
 
 
+    
+class Main:
+    def __init__( self ):
+        debug('init main')
+        self._service_setup()
+        while (not xbmc.abortRequested):
+            xbmc.sleep(1000)
+
+    def _service_setup( self ):
+        debug('installing service')
+        self.Player = MyPlayer(action = self._service_janitor)
+
+    def _service_janitor( self, video_type, expired_videos ):
+        debug('service janitor')
+        # mark as watched
+        debug('%s to clean : %s (%s)' % (video_type, expired_videos['title'], expired_videos['filename']),xbmc.LOGNOTICE)
+        try:
+            cleaner = Cleaner()
+            results, _ = cleaner.clean_all(video_type=video_type, expired_videos=expired_videos)
+            # do janitor
+            debug(results)
+            notify(results)
```

Comment:
If you do a call to ``notify()``, this will call ``debug()`` automatically, so that all notification are also always logged. Doing both is redundant here.